azext_iot/tests/digitaltwins/test_dt_data_history_lifecycle_int.py

```diff
@@ -74,18 +70,17 @@ def tearDownSuite(self):
             )
 
     def test_dt_data_history_adx(self):
-        self.wait_for_capacity(region=ADH_REGION)
+        self.wait_for_capacity()
         instance_name = f"dt{generate_generic_id()}"
         connection_name = f"cn-{generate_generic_id()}"
         table_name = f"tb_{generate_generic_id()}"
         consumer_group = f"cg-{generate_generic_id()}"
         self.add_eventhub_consumer_group(consumer_group=consumer_group)
 
         create_output = self.cmd(
-            "dt create -n {} -g {} -l {} --assign-identity".format(
+            "dt create -n {} -g {}--assign-identity".format(
```

Comment:
"dt create -n {} -g {} --assign-identity"?  (Note space between {} and --assign-identity)

Reply:
Some other spots below as well.

Reply:
fixed